### ISSUE TITLE
Banned from the pool?

### ISSUE THREAD

**Issue by DumelaJames** [MoneroOcean/xmrig#54]
Everytime I try to connect on my main laptop it says, connection refused.

Does this mean I have been banned from the pool on this device? I have 2 other devices that work just fine with the exact same settings

**Comment by Spudz76** [comment]
No, and probably if all of them are on your LAN they have the same IP on the outside so they would all be refused.

Maybe you have the port wrong or some other typo?  Otherwise it could be protection software blocking the outgoing connection sometimes pools get marked as potentially unwanted (because trojan miners) and you have to allow them as exceptions when you are intentionally intending to do mining.  Many other things like Defender will erase the exe's or not allow them to launch and annoying stuff.  And sometimes Windows Update will reset the exceptions just to mess with you.

**Comment by realsetvin** [comment]
> Everytime I try to connect on my main laptop it says, connection refused.
> 
> Does this mean I have been banned from the pool on this device? I have 2 other devices that work just fine with the exact same settings

if this resolved ur issue please close this
